arkane/thermo.py

```diff
@@ -212,6 +212,23 @@ def save(self, outputFile):
                         f.write('\n')
         return chemkin_thermo_string
 
+    def element_count_from_conformer(self):
+        """
+        Get an element count in a dictionary form (e.g.,  {'C': 3, 'H': 8}) from the species.conformer attribute.
+
+        Returns:
+            dict: Element count, keys are element symbols,
+                  values are number of occurrences of the element in the molecule.
+        """
+        element_counts = dict()
+        for number in self.species.conformer.number.value_si:
+            symbol = symbol_by_number[number]
+            if symbol in element_counts:
+                element_counts[symbol] += 1
+            else:
+                element_counts[symbol] = 1
+        return element_counts
+
```

Comment:
It feels like this method might do better as a part of the Conformer class.

Reply:
I looked at it, but this method uses `symbol_by_number()` from Arkane common.py, I wouldn't like to make Arkane a dependency of conformer, not to add the `symbol_by_number()` function to conformer. I say let's leave it as is. Would you agree?

Reply:
When looking at the usages of `symbol_by_number` and the corresponding `get_element_mass`, they all relate in someway to the conformer object's attributes (some of the uses involve converting C to 12 in the Log classes, which is then just converted back in arkane). Ideally, I would think almost any method manipulating the atom's XYZ coordinates and their corresponding masses should be within the Conformer class, since this complies with the principle of encapsulation in object oriented programming. 

Though honestly, it would require some restructuring, and I am fine merging it in, we already have many other poor uses of object oriented programing in the code already.